pom.xml

```diff
@@ -3,8 +3,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-      <!--JENKINS-18537-->
-    <version>1.554.1</version>
+    <version>1.580.1</version>
```

Comment:
As @amuniz mentioned, in order for this plugin to be supported by the workflow plugin, the jenkins version needs to be updated because the workflow plugin requires a higher jenkins minimal version